backend/infrastructures/extract/adapters/strategies.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+
+import pandas as pd
+
+import backend.modules.extract.services.ports.strategies as ex
```

Comment:
It's not clear to me why strategies do have port?

BTW it should be from modules.extarct.services import  strategies.